foreshadow/preparer/preparerstep.py

```diff
@@ -247,29 +247,21 @@ class PreparerStep(BaseEstimator, TransformerMixin):
 
     """
 
-    def __init__(
-        self, use_single_pipeline=False, column_sharer=None, **kwargs
-    ):
+    def __init__(self, column_sharer=None, **kwargs):
         """Set the original pipeline steps internally.
 
         Takes a list of desired SmartTransformer steps and stores them as
         self._steps. Constructs self an sklearn pipeline object.
 
         Args:
             column_sharer: ColumnSharer instance to be shared across all steps.
-            use_single_pipeline: Creates pipelines using SinglePipeline
+            use_single_pipeline: Creates pipelines using SingleInputPipeline
                 class instead of normal Pipelines.  .. #noqa: I102
             **kwargs: kwargs to PIpeline constructor.
 
         """
         self.column_sharer = column_sharer
         self._parallel_process = None
```

Comment:
Should this be a private variable?

Reply:
Yeah to signal to them they shouldn't be changing it.